### ISSUE TITLE
[BUG] Failed to setValue ... "Invalid object for states: /"

### ISSUE THREAD

**Issue by platfowner** [ainblockchain/ain-blockchain#594]
Error logs (see the attached log file for details):
```
2021-09-29T08:29:29.847Z [node-0-8080] error: [CONSENSUS] Previous block's epoch (20361616) is greater than or equal to incoming block's (20361616)
2021-09-29T08:29:35.057Z [node-0-8080] error: [FUNCTIONS]   ==> Failed to setValue on '/staking/consensus/0x004A2550661c8a306207C9dabb279d5701fFD66e/0/expire_at' with error: {"error_message":"Invalid object for states: /","code":101,"bandwidth_gas_amount":1}
2021-09-29T08:29:35.058Z [node-0-8080] error: [CONSENSUS] Failed to execute the proposal tx: {"gas_amount_total":{"bandwidth":{"service":3},"state":{"service":0}},"gas_cost_total":0,"result_list":{"0":{"error_message":"Triggered function call failed","func_results":{"_handleOffenses":{"op_results":{"0":{"path":"/consensus/offense_records/0x004A2550661c8a306207C9dabb279d5701fFD66e","result":{"code":0,"bandwidth_gas_amount":1}},"1":{"path":"/staking/consensus/0x004A2550661c8a306207C9dabb279d5701fFD66e/0/expire_at","result":{"error_message":"Invalid object for states: /","code":101,"bandwidth_gas_amount":1}}},"code":0,"bandwidth_gas_amount":0},"_distributeFee":{"code":0,"bandwidth_gas_amount":0}},"code":105,"bandwidth_gas_amount":1}}}
2021-09-29T08:29:35.720Z [node-0-8080] error: [FUNCTIONS]   ==> Failed to setValue on '/staking/consensus/0x004A2550661c8a306207C9dabb279d5701fFD66e/0/expire_at' with error: {"error_message":"Invalid object for states: /","code":101,"bandwidth_gas_amount":1}
...
```

![Screen Shot 2021-09-29 at 7 13 37 PM](https://user-images.githubusercontent.com/46211617/135249723-8bdeec2d-201a-4671-aeb0-79984c1397ce.jpg)

[node-0-8080-error-2021-09-29.log](https://github.com/ainblockchain/ain-blockchain/files/7250975/node-0-8080-error-2021-09-29.log)

**Comment by platfowner** [comment]
@liayoo  could you take a look?

**Comment by liayoo** [comment]
Fixed by #601
